Website/Website/App_Code/DatabaseTest.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Data.SqlClient;
+using System.Linq;
+using System.Web;
+
+public class DatabaseTest
+{
+    private SqlConnection conn;
+
+    public DatabaseTest()
+    {
+        conn = new SqlConnection("user id=sa;" +
+                                "password=wachtwoord123;" +
+                                "server=localhost;" +
+                                "Trusted_Connection=yes;" +
+                                "database=FestiBase; " +
+                                "connection timeout=30");
+
+        try
+        {
+            conn.Open();
+        }
+        catch (Exception e)
+        {
+            Console.WriteLine(e.ToString());
+        }
+    }
+
+    public string getFestivalbyNumber(int number)
```

Comment:
in volgende versies, deze ook graag in een mooi apart bestand zetten.
Voor nu als proof of concept goed.